CHANGELOG.md

```diff
@@ -1,5 +1,51 @@
 # Changelog
 
+## 🍕 0.4.0
+
+  - ### ✨ Features
+
+    - **Add config file for configuring include / exclude of files to template - [xortive], [pull/174]**
+        
+        Adds support for the [cargo-generate.toml](https://github.com/ashleygwilliams/cargo-generate/blob/master/README.md#include--exclude) file, which allows templates to configure which files should be processed, either using a whitelist method (`include`), or a blacklist method (`exclude`). When both `include` and `exclude` are present, `include` will be preferred. This mirrors [similar `include`/`exclude` behavior in `cargo`](https://doc.rust-lang.org/cargo/reference/manifest.html#the-exclude-and-include-fields-optional), and uses some of the same code.
+
+        ```toml
```

Comment:
let's show a valid example - this example is invalid which is confusing! so i'd add one that is include, one that is exclude, and then show the invalid one and say how it fails